process-services/latest/support/index.md

```diff
@@ -57,5 +56,8 @@ Choose a combination of products to build your own Supported Stack. If anything
 | **Applications** | |
 | Alfresco Digital Workspace 4.3 | Requires Alfresco Process Services 2.4.2. |
 | | |
+| **Services** | |
+| Alfresco Content Services 23.x |  |
+| | |
```

Comment:
Minor comment: please move these lines for **Services** so they are above the **Integrations** section. This should match the order of what's published in the Docs site.

Reply:
Hi Adelaide, can you check whether the changes made are correct?